src/main/java/org/terasology/potions/system/DrinkPotionAuthoritySystem.java

```diff
@@ -217,6 +217,9 @@ public void onPotionWithoutGenomeConsumed(DrinkPotionEvent event, EntityRef ref)
                 case PotionCommonEffects.EXPLOSIVE:
                     herbEffect = new ExplosiveEffect();
                     break;
+				case PotionCommonEffects.NO_VISIBILITY:
+                    herbEffect = new NoVisibilityEffect();
```

Comment:
Note that this class does not appear to be part of the PR. Forget to `git add` it? :-)